js/spreadsheet.js

```diff
@@ -17,6 +17,7 @@ var spreadsheet = function(options) {
   var arrayColumns = [];
   var columnNameCounter = 1; // Counter to anonymous columns names
   var rendered = 0;
+  var isEditing = false;
```

Comment:
Where and how is this variable used?

Reply:
@tonytlwu 
That is my mistake. I was trying to implement the previous solution and forgot to remove it.
Sorry about that.